bin/main.rb

```diff
@@ -0,0 +1,78 @@
+class Player
+  attr_accessor :name, :weapon
+
+  def initialize(name, weapon)
+    @name = name
+    @weapon = weapon
+  end
+end
+
+def draw_board(square_values)
+  puts "     0       1       2   "
+  puts "         |       |       "
+  puts "0    #{square_values[0]}   |   #{square_values[1]}   |   #{square_values[2]}   "
+  puts "         |       |       "
+  puts "  -----------------------"
+  puts "         |       |       "
+  puts "1    #{square_values[3]}   |   #{square_values[4]}   |   #{square_values[5]}   "
+  puts "         |       |       "
+  puts "  -----------------------"
+  puts "         |       |       "
+  puts "2    #{square_values[6]}   |   #{square_values[7]}   |   #{square_values[8]}   "
+  puts "         |       |       "
+end
+
+def get_names
+  puts "Player one name:"
+  name = gets.chomp
+  weapon = "X"
+  $player_one = Player.new(name, weapon)
```

Comment:
**`REQUIRED`**
Please, why are you using **global variables** ?
Sorry but you're not allowed to use it.